src/lib.rs

```diff
@@ -93,6 +126,40 @@ impl Event {
                     friend,
                     message_id,
                 },
+
+            E::ConferenceInvite { friend, kind, ref cookie } =>
```

Comment:
Why `ref cookie`?

Reply:
`Cookie` is a `Vec<u8>` inside.

Reply:
And then you do `cookie.clone().into_bytes()`, but you could do `cookie.into_bytes()` without `ref cookie`

Reply:
`into_bytes()` moves `cookie`

Reply:
> `match *event {`

I see.